tests/global_communication/mpi_listener.hpp

```diff
@@ -97,24 +97,30 @@ class mpi_listener : public testing::EmptyTestEventListener {
     }
     virtual void OnTestCaseEnd(const TestCase& test_case) override {
         printf_helper(
-            "[PASSED %3d; FAILED %3d] of %3d tests in %s\n\n",
+            "    PASSED %d of %d tests in %s\n",
             test_case_tests_-test_case_failures_,
-            test_case_failures_,
             test_case_tests_,
             test_case.name()
         );
+        if (test_case_failures_>0) {
```

Comment:
Only print FAILURE information if there was a failure. The previous version always printed `PASSED xx; FAILED yy` for each test case... and it always looked like there had been a failure. When people see `FAILED` they automatically think that something failed... so I just put the `FAILED` where there are failures. 
Believe me, the reports are much easier to read.